holidays/calendars/hindu.py

```
@@ -191,206 +191,206 @@ class _HinduLunisolar:
+        2090: (NOV, 20),
+        2091: (NOV, 10),
+        2092: (OCT, 30),
+        2093: (NOV, 18),
+        2094: (NOV, 7),
+        2095: (OCT, 27),
+        2096: (NOV, 14),
+        2097: (NOV, 3),
+        2098: (OCT, 23),
+        2099: (NOV, 11),
+        2100: (NOV, 1),
     }
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)